Src/cql-lm/schema/elm/cqlannotations.xsd

```diff
@@ -22,4 +22,61 @@
 			</xs:restriction>
 		</xs:complexContent>
 	</xs:complexType>
+  <xs:simpleType name="ErrorType">
+    <xs:annotation>
+      <xs:documentation>Represents the type of CQL to ELM conversion error.</xs:documentation>
+    </xs:annotation>
+    <xs:restriction base="xs:NMTOKEN">
+      <xs:enumeration value="syntax">
+        <xs:annotation>
+          <xs:documentation>Any kind of syntax error with the CQL.</xs:documentation>
+        </xs:annotation>
+      </xs:enumeration>
+      <xs:enumeration value="include">
+        <xs:annotation>
+          <xs:documentation>An included CQL library could not be loaded.</xs:documentation>
+        </xs:annotation>
+      </xs:enumeration>
+      <xs:enumeration value="semantic">
+        <xs:annotation>
+          <xs:documentation>Any kind of semantic error with the CQL.</xs:documentation>
+        </xs:annotation>
+      </xs:enumeration>
+    </xs:restriction>
+  </xs:simpleType>
+  <xs:complexType name="CqlToElmError">
+    <xs:annotation>
+      <xs:documentation>Represents CQL to ELM conversion errors</xs:documentation>
+    </xs:annotation>
+    <xs:complexContent>
+      <xs:extension base="Locator">
+        <xs:attribute name="library" type="xs:string" use="required">
+          <xs:annotation>
+            <xs:documentation>The line number in the CQL where the error started</xs:documentation>
```

Comment:
This description ("The line number...") doesn't seem to match the attribute name ("library").

If it's really intended to correspond to the "line number", how does this differ from the base `Locator`'s `startLine` attribute?

If it's really intended to be "library", then I'd ask if it's really needed-- as the library will already be specified in the ELM document that contains the annotation (note that `Locator` already assumes current library).  If it's not intended to be the current library, then how does it differ from `targetIncludeLibraryId`?